apps/OboeTester/app/src/main/java/com/mobileer/oboetester/StreamConfigurationView.java

```diff
@@ -94,6 +94,7 @@ public class StreamConfigurationView extends LinearLayout {
     private AudioDeviceSpinner mDeviceSpinner;
     private TextView mActualSessionIdView;
     private CheckBox mRequestAudioEffect;
+    private CheckBox mRequestSessionId;
```

Comment:
Seems like there are other changes in this PR. Can you rebase your change on the latest main?

Reply:
Resolved